payjoin/Cargo.toml

```diff
@@ -36,6 +36,7 @@ reqwest = { version = "0.12", default-features = false, optional = true }
 rustls = { version = "0.22.4", optional = true }
 url = "2.2.2"
 serde_json = "1.0.108"
+bitcoin_hashes = "0.15.0"
```

Comment:
bitcoin 0.32 is using bitcoin_hashes 0.14. Unless we need a feature from 15, we ought not to duplicate the dependency